jdi-light/src/main/java/com/epam/jdi/light/settings/TestProperties.java

```diff
@@ -0,0 +1,263 @@
+import static com.epam.jdi.light.driver.get.RemoteDriver.sauceLabs;
+import static com.epam.jdi.light.driver.get.RemoteDriver.seleniumLocalhost;
+import static com.epam.jdi.light.driver.sauce.SauceSettings.sauceCapabilities;
+import static com.epam.jdi.light.logger.LogLevels.parseLogLevel;
+import static com.epam.jdi.light.logger.Strategy.FAIL;
+import static com.epam.jdi.light.logger.Strategy.addStrategy;
+import static com.epam.jdi.light.logger.Strategy.parseStrategy;
+import static com.epam.jdi.light.settings.JDISettings.COMMON;
+import static com.epam.jdi.light.settings.JDISettings.DRIVER;
+import static com.epam.jdi.light.settings.JDISettings.ELEMENT;
+import static com.epam.jdi.light.settings.JDISettings.LOGS;
+import static com.epam.jdi.light.settings.JDISettings.PAGE;
+import static com.epam.jdi.light.settings.JDISettings.SCREEN;
+import static com.epam.jdi.light.settings.JDISettings.TIMEOUTS;
+import static com.epam.jdi.light.settings.Strategies.JDI;
+import static com.epam.jdi.light.settings.Strategies.SELENIUM;
+import static com.epam.jdi.light.settings.WebSettings.logger;
+import static com.epam.jdi.tools.EnumUtils.getAllEnumValues;
+import static com.epam.jdi.tools.LinqUtils.first;
+import static com.epam.jdi.tools.PrintUtils.print;
+import static java.lang.Boolean.parseBoolean;
+import static java.lang.Integer.parseInt;
+import static java.util.Arrays.asList;
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import static org.openqa.selenium.PageLoadStrategy.EAGER;
+import static org.openqa.selenium.PageLoadStrategy.NONE;
+import static org.openqa.selenium.PageLoadStrategy.NORMAL;
+
+import com.epam.jdi.light.asserts.core.SoftAssert;
+import com.epam.jdi.light.common.ElementArea;
+import com.epam.jdi.light.common.SetTextTypes;
+import com.epam.jdi.light.common.TextTypes;
+import com.epam.jdi.light.common.Timeout;
+import com.epam.jdi.light.common.UseSmartSearch;
+import com.epam.jdi.light.logger.Strategy;
+import com.epam.jdi.tools.LinqUtils;
+import com.epam.jdi.tools.func.JAction;
+import com.epam.jdi.tools.func.JAction1;
+import com.epam.jdi.tools.func.JFunc1;
+import com.epam.jdi.tools.pairs.Pair;
+import org.openqa.selenium.PageLoadStrategy;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TestProperties {
+
+  private static Map<String, Pair<JAction1<String>, JAction>> testsProperties = new HashMap<>();
+  private static final JAction doNothing = ()->{};
+  public static boolean STRICT_SEARCH = true;
+
+  private TestProperties() {
+  }
+
+  public static Map<String, Pair<JAction1<String>, JAction>> getTestsProperties() {
+    return testsProperties;
+  }
+
+  static {
+    testsProperties.put("strategy",
+        new Pair<>(p -> COMMON.strategy = getStrategy(p),
+            COMMON.strategy.action));
+    testsProperties.put("timeout.wait.element",
+        new Pair<>(p -> TIMEOUTS.element = new Timeout(parseInt(p)), doNothing));
```

Comment:
create two methods like this and use it:

public Pair<_type_, _type_> combo(_var_, _var_){       
   return new Pair<>(_var_, _var_));
}

public Pair<_type_, _type_> single(_var_){       
   return combo(_var_, _donothing);
}

Reply:
done